# ros msg for describing a convex hull in 3D

geometry_msgs/Pose pose			# pose anchored at centroid
geometry_msgs/Point[] points			# vertices
geometry_msgs/Vector3 size			# size of hull in x, y, z
